src/Feature/StatusBar/Feature_StatusBar.re

```diff
@@ -40,6 +40,172 @@ module Item = {
   };
 };
 
+module ConfigurationItems = {
+  [@deriving show]
+  type t = {
+    leftItems: list(string),
+    rightItems: list(string),
```

Comment:
It looks like there is a difference between here and the docs - can we use `startItems` and `endItems` instead of `leftItems` and `rightItems`?  I like start/end vs right/left because when we implement a configuration setting for right-to-left languages, `startItems` and `endItems` will be less ambiguous than `right`/`left`.